Dockerfile

```
@@ -0,0 +1,82 @@
+
+# Make Static Dir for essential initialization and run time files
+RUN mkdir /app-static
+COPY go.mod go.sum .env /app-static/
+
+RUN mkdir -p /run/supervisor /var/run/supervisor /var/log/supervisor && \
+    # should think about these permissions for security
+    chown root:root /run/supervisor /var/run/supervisor /var/log/supervisor && \
+    chmod 755 /var/log/supervisor && \
+    chmod 777 /var/run/supervisor && \
+    chmod 777 /run/supervisor
+    mkdir -p /var/lib/postgresql/data && \
```

Hadolint (2.12.0)

[error] unexpected 'm'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)